### PR TITLE
Add new icons and favicons to free.law

### PR DESCRIPTION
**Pull request by mlissner**
This follows on https://github.com/freelawproject/courtlistener/pull/4184 and https://github.com/freelawproject/bigcases2/pull/561, but applies the same principles in nextjs.

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -106,15 +42,11 @@ export default function App({ Component, pageProps }) {
         additionalMetaTags={[
           {
             name: 'msapplication-TileColor',
-            content: '#da532c',
-          },
-          {
-            name: 'msapplication-TileImage',
-            content: '/mstile-144x144.png',
+            content: '#663367',
```
[the remark this answers is not in the document]
Comment (flooie) [reply pages/_app.js:45]: I take it back

**Comment by flooie** [comment]
I have philosophical issues with this one @mlissner

**Comment by mlissner** [comment]
Alright, mind giving this another look, @flooie? I think it should be better, and I hand-optimized the SVG myself. :kiss: :ok_hand:

File: pages/_app.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ export default function App({ Component, pageProps }) {
       <DefaultSeo
         titleTemplate="%s | Free Law Project | Making the legal ecosystem more equitable and competitive."
         defaultTitle="Free Law Project"
-        description="Free Law Project is the leading nonprofit using technology and data to make the legal ecosystem more equitable and competitive. Creators of the CourtListener and RECAP projects."
+        description="Free Law Project is the leading nonprofit using technology and data to make the legal ecosystem more equitable and competitive. Creators of the CourtListener, RECAP, and Bots.law projects."
```
Comment (flooie) [inline pages/_app.js:10]: "the CourtListener" reads funny to me.

**Comment by flooie** [comment]
this looks good to me - but the [favicon-32.png](https://github.com/freelawproject/free.law/pull/193#diff-81d3245fb5d73bf95a65b30640e34cd86c2686a18ef7f89cb8825c98c14d5173) and the favicon.svg look different, where I would expect to see more detail around the lip of the scale - but I only see it in the smaller one.  Otherwise things look good to me.

File: pages/_app.js (hunk 2 above)
Comment (mlissner) [inline pages/_app.js:10]:
```suggestion
        description="Free Law Project is the leading nonprofit using technology and data to make the legal ecosystem more equitable and competitive. Creators of CourtListener, RECAP, and Bots.law."
```

Comment (mlissner) [reply pages/_app.js]: Tweaked, thanks.

**Review by flooie — APPROVED** [review]
👨‍🍳 💋

**Pull request merged**